README.md

```diff
@@ -164,6 +165,31 @@ Since we're using [ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl), we have our
 
 By default we use the flags `--migrate=true --routing=dhtclient ----enable-gc=true` when running the IPFS daemon. They can be changed via the configuration file, which can be easily accessed as mentioned above.
 
+## Troubleshooting
+
+### Does not start on Linux (Debian 10)
+
+Some Linux users may see an error like this:
+
+> The SUID sandbox helper binary was found, but is not configured correctly.
+Rather than run without sandboxing I'm aborting now. You need to make sure that
+chrome-sandbox is owned by root and has mode 4755.
+
+or a very short one, when starting in a terminal:
+
+```console
+$ ipfs-desktop
+$Trace/breakpoint trap
+```
+
+
+**TL;DR** There are two ways of fixing this, pick one that works for you:
```

Comment:
Seems like the first option opens users up to more issues. Any reason why we wouldn't recommend only the solution specific to IPFS?

Reply:
After giving this more thought I removed `kernel.unprivileged_userns_clone` one.
It is not present in regular Linux kernel (Debian added it as part of their hardening patches) and won't work for most people anyway.